packages/excalidraw/element/routing.ts

```diff
@@ -971,9 +971,9 @@ const getGlobalPoint = (
     return initialPoint;
   }
 
-  if (boundElement && fixedPointRatio) {
+  if (boundElement) {
     const fixedGlobalPoint = getGlobalFixedPointForBindableElement(
-      fixedPointRatio,
+      fixedPointRatio || [0, 0],
```

Comment:
this doesn't make much sense to me, but it's what we were effectively doing pre [`4b66f11` (#8299)](https://github.com/excalidraw/excalidraw/pull/8299/commits/4b66f111e7fdd947d4259f36f4ee1a125fde3802)

Reply:
I guess this is resolved in fixing the simple arrow angle bug. Having a backup to make sure it's never undefined is just an extra safety step, it cannot occur anymore.